test/e2e/burial-helpers.js

```diff
@@ -0,0 +1,152 @@
+
+      if (index < data.previousNames.length - 1) {
+        client.click('.usa-button-outline.va-growable-add-btn');
+      }
+    });
+  } else {
+    client.selectYesNo('root_view:serveUnderOtherNames', false);
+  }
+}
+
+function completeBenefitSelection(client, data) {
+  client
+    .clickIf('input[name="root_view:claimedBenefits_burialAllowance"]', data['view:claimedBenefits'].burialAllowance)
+    .clickIf('input[name="root_view:claimedBenefits_plotAllowance"]', data['view:claimedBenefits'].plotAllowance);
+
+  // TODO: Uncomment this when we get file upload working and testable
+  // if (data['view:claimedBenefits'].transportation) {
+  //   client
+  //     .click('input[name="root_view:claimedBenefits_transportation"]')
+  //     .fill('input[name="root_view:claimedBenefits_amountIncurred"]', data['view:claimedBenefits'].amountIncurred);
+  // }
+}
+
+function completeBurialAllowance(client, data) {
+  client.selectRadio('root_burialAllowanceRequested', data.burialAllowanceRequested);
+
+  if (data.burialAllowanceRequested === 'vaMC') {
+    client.fill('input[name="root_burialCost"]', data.burialCost);
+  }
+
+  if (data.relationship.type === 'spouse') {
+    client.selectYesNo('root_previouslyReceivedAllowance', data.previouslyReceivedAllowance);
+  } else if (data.relationship.type === 'other') {
+    client.selectYesNo('root_benefitsUnclaimedRemains', data.benefitsUnclaimedRemains);
+  }
+}
+
+function completePlotAllowance(client, data) {
+  client
+    .fill('input[name="root_placeOfRemains"]', data.placeOfRemains)
+    .selectYesNo('root_federalCemetery', data.federalCemetery);
+
+  if (!data.federalCemetery) {
+    client.selectYesNo('root_stateCemetery', data.stateCemetery);
+  }
+
+  client.selectYesNo('root_govtContributions', data.govtContributions);
+  if (data.govtContributions) {
+    client.fill('input[name="root_amountGovtContribution"]', data.amountGovtContribution);
+  }
+}
+
+function completeClaimantContactInformation(client, data) {
+  if (!data.relationship['view:isEntity']) {
+    client.fill('input[name="root_officialPosition"]', data.officialPosition);
+  }
+  client
+    .fillAddress('root_claimantAddress', data.claimantAddress)
+    .fill('input[name="root_claimantEmail"]', data.claimantEmail)
+    .fill('input[name="root_claimantPhone"]', data.claimantPhone);
+}
+
+
+function initApplicationSubmitMock() {
+  mock(null, {
+    path: '/v0/burial_claims',
+    verb: 'post',
+    value: {
+      data: {
+        attributes: {
+          regionalOffice: [],
+          confirmationNumber: '123fake-submission-id-567',
+          submittedAt: '2016-05-16'
+        }
+      }
+    }
+  });
+}
+
+module.exports = {
+  completeClaimantInformation,
+  completeVeteranInformation,
+  completeBurialInformation,
+  completeServicePeriods,
+  completePreviousNames,
+  completeBenefitSelection,
+  completeBurialAllowance,
+  completePlotAllowance,
+  completeClaimantContactInformation,
+
```

Comment:
Yeah, I could take or leave it. I just had it to separate the pages from the mocks while I was adding all the page-filling functions.